website/docs/how-to/how-to-create-feature-toggles.md

```diff
@@ -1,23 +1,29 @@
 ---
-title: How to create a feature toggle
+title: How to create a feature flag
```

Comment:
Ooh, you wanna be very careful with this one. I think this will cause a URL change, so we should also add a redirect rule for this.

Reply:
It didn't on my local - just went back and checked again and it's ok. 
I think the paths all live in sidebar.js and I was deliberate about not changing this one.

Reply:
Got it. In that case, it probably uses the file name instead. And you're right; the url is the same on the preview too. Thanks for checking.